builder/steps/gen-dockerfile/contents/gulpfile.js

```diff
@@ -94,7 +94,8 @@ gulp.task('test.compile', ['compile'], function() {
 });
 
 gulp.task('test.unit', ['test.compile'], function() {
-  return gulp.src([outDir + '/test/**/*.js']).pipe(mocha({reporter: 'spec'}));
+  return gulp.src([outDir + '/test/**/*.js'])
+      .pipe(mocha({reporter: 'spec', require: 'source-map-support/register'}));
```

Comment:
It is not related indeed. It makes sure that mocha stack traces show TS rather than JS stacks.